tests/unit/mixins/sl-input-based-test.js

```diff
@@ -0,0 +1,200 @@
+import Ember from 'ember';
+import mixinUnderTest from 'sl-ember-components/mixins/sl-input-based';
+
+module( 'Unit | Mixin | sl input based' );
+
+test( 'Successfully mixed', function( assert ) {
+    let testObject = Ember.Object.extend( mixinUnderTest );
+    let subject = testObject.create();
+
+    assert.ok( subject );
+});
+
+test( 'Disabled, optional, readonly, and required are false by default', function( assert ) {

```

Comment:
Change the name of this test to something more generic, such as "Default values are set to expected values"